velox/core/PlanNode.h

```diff
@@ -246,6 +249,50 @@ class ValuesNode : public PlanNode {
   const bool parallelizable_;
 };
 
+class ArrowStreamNode : public PlanNode {
+ public:
+  ArrowStreamNode(
+      const PlanNodeId& id,
+      const RowTypePtr& outputType,
+      std::shared_ptr<ArrowArrayStream> arrowStream,
+      memory::MemoryPool* pool)
+      : PlanNode(id),
+        outputType_(outputType),
+        arrowStream_(std::move(arrowStream)),
+        pool_(pool) {
+    VELOX_CHECK(arrowStream_ != nullptr);
+  }
+
+  const RowTypePtr& outputType() const override {
+    return outputType_;
+  }
+
+  const std::vector<PlanNodePtr>& sources() const override;
+
+  bool requiresSplits() const override {
```

Comment:
I don't see this operator using splits. This method should return false, no?

Reply:
You're right. Fixed.